src/files/BrsFile.Class.spec.ts

```diff
@@ -417,6 +417,61 @@ describe('BrsFile BrighterScript classes', () => {
             `, undefined, 'source/main.bs');
         });
 
+        it('transpiles super when not the first statement', () => {
+            testTranspile(`
+                class Creature
+                    sub new(name as string)
+                    end sub
+                    function sayHello(text)
+                    ? text
+                    end function
+                end class
+
+                class Duck extends Creature
+                    override function sayHello(text)
+                        text = "The duck says " + text
+                        if text <> invalid
+                            super.sayHello(text)
+                        end if
+                    end function
+                end class
+            `, `
+        function __Creature_builder()
```

Comment:
Fix the indentation for this entire string template so it matches the one above. Then pass 'trim,' instead of undefined as the third parameter to testTranspile